test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -331,3 +338,22 @@ def toggle_pin_message_at_index(self, message_index: int):
     def check_pin_count(self, wanted_pin_count: int):
         pin_text_obj = wait_and_get_obj(CommunityScreenComponents.PIN_TEXT.value)
         verify_equals(pin_text_obj.text, wanted_pin_count)
+
+    def block_member(self, member: str):
+        right_click_obj_by_wildcards_name(CommunityUserListPanel.COMMUNITY_MEMBER_PLACEHOLDER.value, member)
+        block_user_obj = get_obj(CommunityMembersContextMenu.BLOCK_MENU_ITEM.value)
+        hover_obj(block_user_obj)
```

Comment:
like this trick

Reply:
I'm not sure why you need to hover the object before clicking on it?
Possible to block and unblock users

Reply:
as @MishkaRogachev and @endulab mentioned - sometimes it doesn't work. :)